satellite/gracefulexit/endpoint.go

```diff
@@ -235,6 +235,13 @@ func (endpoint *Endpoint) doProcess(stream processStream) (err error) {
 		}()
 
 		for range ticker.C {
```

Comment:
The context cancellation must wait concurrently with ticker not a separate thing.

Reply:
Does that mean when context cancelled we should call `ticker.Stop()`?

Reply:
https://github.com/storj/storj/pull/3394

Reply:
@VinozzZ, no it means to listen for ctx canceled in a select statement along with `ticker.C`